nym-wallet/src/context/bonding.tsx

```diff
@@ -282,14 +296,15 @@ export const BondingContextProvider = ({ children }: { children?: React.ReactNod
         const data = await getGatewayBondDetails();
         if (data) {
           const nodeDescription = await getNodeDescription(data.gateway.host, data.gateway.clients_port);
-
+          const routingScore = await getGatewayReportDetails(data.gateway.identity_key);
           setBondedNode({
             name: nodeDescription?.name,
             identityKey: data.gateway.identity_key,
             ip: data.gateway.host,
             location: data.gateway.location,
             bond: data.pledge_amount,
             proxy: data.proxy,
+            routingScore,
```

Comment:
`routingScore` is not typed to be nullable (aka `| undefined`) in `TBondedGateway`. But if  `getGatewayReportDetails` fails for some reason, `routingScore` will be `undefined`, and potentially cause some client crash since it's an object with properties, and accessing a property on `undefined` value will cause a runtime crash.

Reply:
Good point. Will update this.